robottelo/ui/errata.py

```diff
@@ -131,13 +131,30 @@ def check_errata_details(self, errata_id, parameter_list=None,
                 'errata',
                 (parameter_name.lower()).replace(' ', '_')
             ))
-            actual_text = self.wait_until_element(locators[param_locator]).text
-            if parameter_value not in actual_text:
-                raise UIError(
-                    'Actual text for "{0}" parameter is "{1}", but it is'
-                    ' expected to have "{2}"'.format(
-                        parameter_name, actual_text, parameter_value)
-                )
+            if isinstance(parameter_value, set):
+                if self.wait_until_element_exists(locators[param_locator]):
+                    actual_text_set = {
+                        element.text.strip()
+                        for element in self.find_elements(
+                            locators[param_locator])
+                        }
+                else:
+                    actual_text_set = set()
```

Comment:
How about moving initialization of actual_text_st to line 135 so we can get rid of the else?

```python
actual_text_set=set()
if self wait
     ...
     ....
# no else here anymore